src/plugins/git/demoData/exampleRepo.js

```diff
@@ -18,8 +37,40 @@ export function createExampleRepo(intoDirectory: string): RepositoryInfo {
 
   git.exec(["init"]);
 
-  git.writeAndStage("README.txt", "Amazing physics going on...\n");
+  git.writeAndStage(
+    "README.txt",
+    [
+      "example-git\n",
+      "-----------\n\n",
+      "This repository provides example data for the SourceCred Git plugin.\n",
+      "Pay no attention to the contents behind the curtain.\n",
+    ].join()
+  );
   git.deterministicCommit("Initial commit");
+
+  git.writeAndStage("science.txt", "Amazing physics going on...\n");
+  commits.push(git.head());
```

Comment:
instead of manually managing the commits array (which allows desync, as seen here where we seem to be missing a commit?), lets have one function that commits and updates the array

Reply:
Good idea. Done.